boards/x86: Use right GPIO names for CC2520

tbursztyka opened the pull request:
GPIO_QMSI_<0/1>_NAME are now generated through DTS. Until CC2520
settings are generated through Kconfig, let's use direct gpio names
instead.

Fixes #8086

Signed-off-by: Tomasz Bursztyka <tomasz.bursztyka@linux.intel.com>

Commit:
boards/x86: Use right GPIO names for CC2520

GPIO_QMSI_<0/1>_NAME are now generated through DTS. Until CC2520
settings are generated through DTS, let's use direct gpio names
instead.

Fixes #8086

Signed-off-by: Tomasz Bursztyka <tomasz.bursztyka@linux.intel.com>
Signed-off-by: Anas Nashif <anas.nashif@intel.com>

MaureenHelm commented:
> GPIO_QMSI_<0/1>_NAME are now generated through DTS. Until CC2520
settings are generated through Kconfig, let's use direct gpio names
instead.

Do you mean "Until CC2520 settings are generated through **DTS**"?

nashif reviewed (approved):
will have a small followup to treat it like a test in hw automation.

nashif committed:
tests: ieee802154/crypto: use console harness

Use console harness to detect PASS/FAIL. This test should be moved to
ztest at some point.

Signed-off-by: Anas Nashif <anas.nashif@intel.com>

nashif commented:
> Do you mean "Until CC2520 settings are generated through DTS"?

fixed that for @tbursztyka  :)

and added another commit to fix #8086 proper
